WordPressKit/BlogJetpackSettingsServiceRemote.swift

```diff
@@ -231,7 +231,7 @@ private extension BlogJetpackSettingsServiceRemote {
 
 public extension BlogJetpackSettingsServiceRemote {
 
-    public enum Keys {
+    enum Keys {
```

Comment:
I'm presuming these were removed because it's duplicate visibility given `BlogJetpackSettingsServiceRemote` is `public`?

Reply:
Yep. This was the warning:

![image](https://user-images.githubusercontent.com/1773641/55249546-37dd1400-5244-11e9-843b-34ce765e05cd.png)